src/components/common/Tabs/TabItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+
+interface TabProps {
+  title: string
+  readonly id: number
+  readonly isActive: boolean
+  readonly activeColor?: string
```

Comment:
Faaancy :+1: 
@W3stside , @anxolin why aren't we using `readonly` more, we totally should

Reply:
/shrug